sys/kern/subr_pcpu.c

```diff
@@ -343,9 +347,9 @@ DB_SHOW_COMMAND(dpcpu_off, db_show_dpcpu_off)
 	int id;
 
 	CPU_FOREACH(id) {
-		db_printf("dpcpu_off[%2d] = 0x%jx (+ DPCPU_START = %p)\n",
-		    id, (uintmax_t)dpcpu_off[id],
-		    (void *)(uintptr_t)(dpcpu_off[id] + DPCPU_START));
+		db_printf("dpcpu_off[%2d] = %p (+ DPCPU_BIAS = %p)\n",
+		    id, (void *)dpcpu_off[id],
+		    (void *)(dpcpu_off[id] + DPCPU_BIAS));
```

Comment:
Eh, this is probably wrong.  I might fix this to be more like vnet where we print both pointers on separate lines.